README.md

```diff
@@ -84,6 +84,10 @@ Usage: <converter.py> -i INPUT_FILE(S) -o OUTPUT_FILE -d YYYYMMDDHH
   * `hgodas_adt2ioda.py`
   * `hgodas_insitu2ioda.py`
   * `hgodas_sst2ioda.py`  
+
+* AOML hurricane glider - preprocessed, QCd S/T from AOML hurricane glider for 2020 hurricane track analysis project
+  * glider2ioda.py`
```

Comment:
```suggestion
  * `glider2ioda.py`
```

Reply:
Three things:

1, if I run ctest under {build_dir}/ioda-converters/test, all ctests including glider but one (ncep_bufr) failed:

 5/24 Test  #5: test_iodaconv_godae_prof ..........   Passed    0.50 sec
      Start  6: test_iodaconv_godae_ship
 6/24 Test  #6: test_iodaconv_godae_ship ..........   Passed    0.49 sec
      Start  7: test_iodaconv_godae_trak
 7/24 Test  #7: test_iodaconv_godae_trak ..........   Passed    0.43 sec
      Start  8: test_iodaconv_hgodas_insitu
 8/24 Test  #8: test_iodaconv_hgodas_insitu .......   Passed    0.37 sec
      Start  9: test_iodaconv_glider
 **9/24 Test  #9: test_iodaconv_glider ..............   Passed    0.39 sec
      Start 10: test_iodaconv_hgodas_sst**
10/24 Test #10: test_iodaconv_hgodas_sst ..........   Passed    0.40 sec
      Start 11: test_iodaconv_argoclim
11/24 Test #11: test_iodaconv_argoclim ............   Passed    0.40 sec
      Start 12: test_iodaconv_cryosat2
12/24 Test #12: test_iodaconv_cryosat2 ............   Passed    0.59 sec
      Start 13: test_iodaconv_viirs_jpss1_oc_l2
96% tests passed, 1 tests failed out of 24

Total Test time (real) =  11.05 sec

The following tests FAILED:
         **15 - test_iodaconv_ncep_bufr (Failed)**

2. however, if i ran ctest under {build_dir}/ioda-converters, then 8 failed out of 32 (coding norm error), but **glider still passed,** so my question is: so seems coding norm error does not apply to glider but the other 8 of ctests, is this something that I should work on since glider passed?

14/31 Test #14: test_iodaconv_godae_trak .............   Passed    0.49 sec
      Start 15: test_iodaconv_hgodas_insitu
15/31 Test #15: test_iodaconv_hgodas_insitu ..........   Passed    0.45 sec
      **Start 16: test_iodaconv_glider
16/31 Test #16: test_iodaconv_glider .................   Passed    0.46 sec**
      Start 17: test_iodaconv_hgodas_sst
17/31 Test #17: test_iodaconv_hgodas_sst .............   Passed    0.43 sec

74% tests passed, 8 tests failed out of 31

Total Test time (real) =  15.08 sec

The following tests FAILED:
          1 - iodaconv_chem_coding_norms (Failed)
          2 - iodaconv_gsi-ncdiag_coding_norms (Failed)
          3 - iodaconv_lib-python_coding_norms (Failed)
          4 - iodaconv_marine_coding_norms (Failed)
          5 - iodaconv_ncep_coding_norms (Failed)
          6 - iodaconv_ssec_coding_norms (Failed)
          7 - iodaconv_wrfda-ncdiag_coding_norms (Failed)
         22 - test_iodaconv_ncep_bufr (Failed)

3. If you still need me to work on coding norm (for the other failed tests), can you provide instruction? how am I supposed to test coding norm conflict and how to fix, cause I have no tool of testing nor fixing .

4. Last, I removed files you asked and renamed glider input file.